samples/embedded-map/src/components/EmbeddedMap/EmbeddedMapModel.ts

```diff
@@ -136,37 +219,112 @@ export default class EmbeddedMapModel extends ComponentModelBase {
         });
     }
 
+    private _handleViewerUpdate(event: LocationMarkerEvent): void {
+        this._lastMarkerUpdate = event;
+    }
+
+    private async _moveCloseToPosition(
+        latitude: number,
+        longitude: number
+    ): Promise<void> {
+        if (
+            this._currentPosition?.latitude === latitude &&
+            this._currentPosition?.longitude === longitude
+        ) {
+            return;
+        }
+
+        const url = `${this.imageQueryUrl}?client_id=${this.mapillaryKey}&closeto=${longitude},${latitude}&radius=${this.searchRadius}`;
+
+        const response = await fetch(url, {
```

Comment:
Did Mapillary's `moveCloseTo()` not work for us?

Reply:
When testing this with GVH I found that it required values that were too close, and the search radius is not configurable. The user should be able to drop the marker pretty much wherever in a reasonable city grid and snap to the nearest street or poi. The configurability can also be used to adapt the tool to working with more sparse data at different scales than an urban core.